lib/registration.pm

```diff
@@ -697,7 +697,9 @@ sub handle_scc_popups {
                 next;
             }
             elsif (match_has_tag("remove-repository")) {
+                wait_still_screen 10;
                 send_key $cmd{next};
+                wait_still_screen 10;
```

Comment:
Can it use send_key_until_needlematch?

Reply:
I don't think so. It failed for after send next key, in the assert loop it will assert remove-repository again since the page change is slow, then cause send one more next key which is the root cause of the failure.